explainaboard/info.py

```diff
@@ -47,23 +40,23 @@ class PaperInfo:
 
 @dataclass
 class Performance:
-    metric_name: str = None
```

Comment:
`metric_name` and `value` are not actually optional.

Reply:
Fixed: bc84d7b